site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -364,7 +400,10 @@ describe('Test cases revolving around polls functionality', () => {
         cy.logout();
         cy.login();
         cy.visit(['sample', 'polls']);
-        cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().click();
+        cy.wait(8000);
```

Comment:
Is this 8-second wait as tight a bound as it can be?

Reply:
I changed it to 6000 seconds, it might cause it to be more flaky I'll run the cypress test multiple times to test

Reply:
Tested three times times through github and locally tested as well. Seems to work